src/pages/home/report/ReportActionItemSingle.tsx

```diff
@@ -289,6 +326,78 @@ function ReportActionItemSingle({
     const statusText = status?.text ?? '';
     const statusTooltipText = formattedDate ? `${statusText ? `${statusText} ` : ''}(${formattedDate})` : statusText;
 
+
+    const getHeading = useCallback(() => {
+        return () => {
+            if (displayAllActors && personArray.length === 2 && isReportPreviewAction) {
+                return (
+                    <View style={[styles.flex1, styles.flexRow, styles.overflowHidden]}>
+                        <ReportActionItemFragment
+                            style={[styles.flexShrink1, styles.flexBasis0]}
+                            key={`person-${action?.reportActionID}-${0}`}
+                            accountID={Number(icon.id)}
+                            fragment={{...personArray.at(0), type: 'TEXT', text: icon?.name ?? ''}}
+                            // delegateAccountID={Number(11111)}
+                            isSingleLine
+                            actorIcon={icon}
+                            moderationDecision={getReportActionMessage(action)?.moderationDecision?.decision}
+                        />
+                        <Text
+                            numberOfLines={1}
```

Comment:
im using the method I used in the original PR but I probably will clean up the icon logic in this file because it's... well... terrifying.